dev/source/docs/building-setup-linux.rst

```diff
@@ -10,6 +10,28 @@ This article shows how to setup your build environment on Linux/Ubuntu machines.
 Setup on Ubuntu
 ===============
 
+Install some required packages
+------------------------------
+
+If you are on a debian based system (such as Ubuntu or Mint), we provide `a script <https://github.com/ArduPilot/ardupilot/blob/master/Tools/environment_install/install-prereqs-ubuntu.sh>`__ that will do it for you. From ardupilot directory :
```

Comment:
I am not sure of this change as we need git to get the install script ...

Reply:
Absolutely! will fix...but still needs a little re-org...will redo and ask for a re-review...thanks!

Reply:
Think I have the flow correct now....Unbuntu with git/clone first, other Linux, MacOS,Docker